tasmota/language/it-IT.h

```diff
@@ -120,55 +120,55 @@
 #define D_MULTI_PRESS "multi-pressione"
 #define D_NOISE "Rumore"
 #define D_NONE "Nessuno"
-#define D_OFF "Off"
+#define D_OFF "OFF"
 #define D_OFFLINE "Offline"
-#define D_OK "Ok"
-#define D_ON "On"
+#define D_OK "OK"
+#define D_ON "ON"
 #define D_ONLINE "Online"
 #define D_PASSWORD "Password"
 #define D_PORT "Porta"
 #define D_POWER_FACTOR "Fattore di potenza"
 #define D_POWERUSAGE "Potenza"
-#define D_POWERUSAGE_ACTIVE "Potenza Attiva"
-#define D_POWERUSAGE_APPARENT "Potenza Apparente"
-#define D_POWERUSAGE_REACTIVE "Potenza Reattiva"
+#define D_POWERUSAGE_ACTIVE "Potenza attiva"
+#define D_POWERUSAGE_APPARENT "Potenza apparente"
+#define D_POWERUSAGE_REACTIVE "Potenza reattiva"
 #define D_PRESSURE "Pressione"
 #define D_PRESSUREATSEALEVEL "Pressione al livello del mare"
-#define D_PROGRAM_FLASH_SIZE "Dimensione Flash Programma"
-#define D_PROGRAM_SIZE "Dimensione Programma"
+#define D_PROGRAM_FLASH_SIZE "Dimensione flash programma"
+#define D_PROGRAM_SIZE "Dimensione programma"
 #define D_PROJECT "Progetto"
-#define D_RAIN "Rain"
-#define D_RANGE "Range"
+#define D_RAIN "Pioggia"
+#define D_RANGE "Intervallo"
 #define D_RECEIVED "Ricevuto"
 #define D_RESTART "Riavvio"
-#define D_RESTARTING "Riavviando"
-#define D_RESTART_REASON "Causa Riavvio"
+#define D_RESTARTING "Riavvio"
+#define D_RESTART_REASON "Causa riavvio"
 #define D_RESTORE "ripristino"
 #define D_RETAINED "salvato"
-#define D_RULE "Rule"
+#define D_RULE "Regola"
 #define D_SAVE "Salva"
 #define D_SENSOR "Sensore"
-#define D_SSID "SSId"
-#define D_START "Start"
+#define D_SSID "SSID"
+#define D_START "Avvia"
 #define D_STD_TIME "STD"
 #define D_STOP "Stop"
-#define D_SUBNET_MASK "Maschera Subnet"
-#define D_SUBSCRIBE_TO "Sottoscrivi a"
-#define D_UNSUBSCRIBE_FROM "Cancella da"
-#define D_SUCCESSFUL "Riuscito"
+#define D_SUBNET_MASK "Maschera sottorete"
+#define D_SUBSCRIBE_TO "Abbonati a"
+#define D_UNSUBSCRIBE_FROM "Rimuovi abbonamento da"
+#define D_SUCCESSFUL "Completato"
 #define D_SUNRISE "Alba"
 #define D_SUNSET "Tramonto"
 #define D_TEMPERATURE "Temperatura"
 #define D_TO "a"
-#define D_TOGGLE "Toggle"
+#define D_TOGGLE "ON/OFF"
```

Comment:
"To toggle" significa commutare. ON/OFF non é corretto.

Reply:
Toogle = Inglese
Commuta è in Italiano ma per il 99,99% degli Italiani ON/Off è più chiaro di "Commuta"

Reply:
Si, a pensarci bene hai ragione. Lascia ON/OFF.

Reply:
toggle: "cambia stato", ma è veramente troppo lungo.